samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/Model.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DatabaseIntegration 
+{
+    public class AdultCensusContext : DbContext
+    {
+        //public DbSet<Blog> Blogs { get; set; }
+        //public DbSet<Post> Posts { get; set; }
+        public DbSet<AdultCensus> AdultCensus { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlite("Data Source=mlexample.db");
+        }
+    }
+    public class AdultCensus
```

Comment:
NIT: if you rename this class to Adult, AdultId will be recognized as a database generated key by convention (because it matches the naming pattern classname + "Id") and the attributes won't be necessary.